jjb/egeria/egeria.yaml

```diff
@@ -21,9 +21,9 @@
     ossrh-profile-id: '{ossrh_profile_id}'
 
 - project:
-    name: egeria-connector-ibm-igc
-    project-name: egeria-connector-ibm-igc
-    project: egeria-connector-ibm-igc
+    name: odpi-igc-connector
+    project-name: odpi-igc-connector
+    project: odpi-igc-connector
```

Comment:
This GitHub project does not exist. Additionally, since this is a separate repo, it should be living in a separate document.

Our standard repository layout should be:

ci-management/jjb/<repo>/{<repo>|<support_config>}.yaml